repo/packages/A/artifactory-primary/0/resource.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images": {
+    "icon-small": "https://raw.githubusercontent.com/jainishshah17/Images/master/jfrog_sml.png",
+    "icon-medium": "https://raw.githubusercontent.com/jainishshah17/Images/master/jfrog_med.png",
+    "icon-large": "https://raw.githubusercontent.com/jainishshah17/Images/master/jfrog_lrg.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "e183c36bdb4b": "jfrog-docker-reg2.bintray.io/jfrog/artifactory-pro:4.7.6",
```

Comment:
You can use more descriptive keys than the Git SHA for each of these images (perhaps something like `artifactory-pro-4.7.6` and `artifactory-ha-primary-4.7.6`). This is easier to update and less likely to become out of date.